### DISCUSSION TITLE
Auto rewind doesn't work when pausing with headphone cord pause button

### DISCUSSION THREAD

**Discussion by dskrad** [PaulWoitaschek/Voice#1894] [Ideas]
I set auto rewind setting to 4s. This works when I pause AND resume in the app or using the notification shade. However, the auto rewind does NOT work when pausing or resuming from a headphone play/pause toggle button press.

**Comment by HighElder** [comment]
I cannot reproduce this issue with my blutooth headphones. What hardware do you use?

**Comment by dskrad** [comment]
Using a pair of earbuds that have a play/pause button inline.
